scalafmt-tests/src/test/resources/scala3/Given.stat

```diff
@@ -0,0 +1,260 @@
+
+<<< given with body
+maxColumn = 40
+===
+object Order {
+given intOrd: Ord[Int] with {def compare(x: Int, y: Int) = { if (x < y) -1 else if (x > y) +1 else 0 }}
+}
+>>>
+object Order {
+  given intOrd: Ord[Int] with {
+    def compare(x: Int, y: Int) = {
+      if (x < y) -1
+      else if (x > y) +1
+      else 0
+    }
+  }
+}
+<<< given with body and multi parents
+maxColumn = 30
+===
+given intOrd: Ord[Int] with Eq[T] with Sort[T] with {def compare(x: Int, y: Int) = {if (x < y) -1 else if (x > y) +1 else 0 }}
+>>>
+given intOrd: Ord[Int]
+  with Eq[T]
+  with Sort[T]
+  with {
+    def compare(
+        x: Int,
+        y: Int
+    ) = {
+      if (x < y) -1
+      else if (x > y) +1
+      else 0
+    }
+  }
+<<< given with body and multi parents single line
+given intOrd: Ord[Int] with Eq[T] with Sort[T] with {}
+>>>
+given intOrd: Ord[Int] with Eq[T] with Sort[T] with {}
+<<< using and type param
+maxColumn = 30
+===
+given listOrd[T](using ord: Ord[T]): Alpha with Beta with {
+  def a = ???
+}
+>>>
+given listOrd[T](using
+    ord: Ord[T]
+): Alpha with Beta with {
+  def a = ???
+}
+<<< anonymous given
+maxColumn = 30
+===
+given [T](using ord: Ord[T]): Alpha with Beta with {
+  def a = ???
+}
+>>>
+given [T](using
+    ord: Ord[T]
+): Alpha with Beta with {
+  def a = ???
+}
+<<< anonymous given multi parents
+maxColumn = 30
+===
+given [T](using ord: Ord[T]): Alpha with Beta with Gamma with Epsilon with {
+  def a = ???
+}
+>>>
+given [T](using
+    ord: Ord[T]
+): Alpha
+  with Beta
+  with Gamma
+  with Epsilon
+  with {
+    def a = ???
+  }
+<<< abstract given
+object Order {
+given listOrd  [T](using  ord: Ord[T])  : Eq[T]
+}
+>>>
+object Order {
+  given listOrd[T](using ord: Ord[T]): Eq[T]
+}
+<<< abstract given long
+maxColumn = 25
+===
+given listOrd  [T](using  ord: Ord[T])  : Eq[T]
+>>>
+given listOrd[T](using
+    ord: Ord[T]
+): Eq[T]
+<<< alias given
+object O { 
+  given   global: ExecutionContext =   ForkJoinPool()}
+>>>
+object O {
+  given global: ExecutionContext = ForkJoinPool()
+}
+<<< alias given long
+maxColumn = 35
+===
+given global: ExecutionContext = ForkJoinPool()
+>>>
+given global: ExecutionContext =
+  ForkJoinPool()
+<<< alias abstract given
+given   ExecutionContext[  NoActualParameterIsNormallyHere] =   ForkJoinPool()
+>>>
+given ExecutionContext[NoActualParameterIsNormallyHere] = ForkJoinPool()
+<<< alias abstract given long
+maxColumn = 25
+===
+given   ExecutionContext[  NoActualParameterIsNormallyHere] =   ForkJoinPool()
+>>>
+given ExecutionContext[
+  NoActualParameterIsNormallyHere
+] = ForkJoinPool()
+<<< parent constructors `never`
+binPack.parentConstructors = Never
+maxColumn = 40
+===
+given a:A with Alpha with Beta with Gamma with Delta with Epsilon with {
+  def a = ???
+}
+>>>
+given a: A
+  with Alpha
+  with Beta
+  with Gamma
+  with Delta
+  with Epsilon
+  with {
+    def a = ???
```

Comment:
just a question: is this better (or more consistent with scala3's significant indentation) than if we had reduced indentation inside `{}`?

Reply:
Actually, let me check. Maybe it's enough to just have the same level of indentation as 'with'.

Reply:
Removed the increased indentation. It's readable enough without it. What to you think?

Reply:
looks good to me